cmd/validator/wallet/create.go

```diff
@@ -0,0 +1,171 @@
+package wallet
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/manifoldco/promptui"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v3/cmd/validator/flags"
+	"github.com/prysmaticlabs/prysm/v3/io/prompt"
+	"github.com/prysmaticlabs/prysm/v3/validator/accounts"
+	"github.com/prysmaticlabs/prysm/v3/validator/accounts/userprompt"
+	"github.com/prysmaticlabs/prysm/v3/validator/accounts/wallet"
+	"github.com/prysmaticlabs/prysm/v3/validator/keymanager"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	// #nosec G101 -- Not sensitive data
+	newMnemonicPassphraseYesNoText = "(Advanced) Do you want to setup a '25th word' passphrase for your mnemonic? [y/n]"
+	// #nosec G101 -- Not sensitive data
+	newMnemonicPassphrasePromptText = "(Advanced) Setup a passphrase '25th word' for your mnemonic " +
+		"(WARNING: You cannot recover your keys from your mnemonic if you forget this passphrase!)"
+)
+
+func walletCreate(c *cli.Context) error {
+	keymanagerKind, err := inputKeymanagerKind(c)
+	if err != nil {
+		return err
+	}
+
+	opts, err := ExtractWalletCreationConfigFromCli(c, keymanagerKind)
+	if err != nil {
+		return err
+	}
+
+	acc, err := accounts.NewCLIManager(opts...)
+	if err != nil {
+		return err
+	}
+	if _, err := acc.WalletCreate(c.Context); err != nil {
+		return errors.Wrap(err, "could not create wallet")
+	}
+	return nil
+}
+
+// ExtractWalletCreationConfigFromCli prompts the user for wallet creation input.
+func ExtractWalletCreationConfigFromCli(cliCtx *cli.Context, keymanagerKind keymanager.Kind) ([]accounts.Option, error) {
```

Comment:
is there a better name for this now that you have refactored it to return account options?

Reply:
nice, good point! I changed it to `ConstructCLIManagerOpts`! 

thanks!